commands/azureCommands/acr-logs-utils/tableViewManager.ts

```diff
@@ -14,13 +15,11 @@ export class LogTableWebview {
         this.panel = vscode.window.createWebviewPanel('log Viewer', webviewName, vscode.ViewColumn.One, { enableScripts: true, retainContextWhenHidden: true });
 
         //Get path to resource on disk
-        let extensionPath = vscode.extensions.getExtension("PeterJausovec.vscode-docker").extensionPath;
-        const scriptFile = vscode.Uri.file(path.join(extensionPath, 'commands', 'azureCommands', 'acr-logs-utils', 'logScripts.js')).with({ scheme: 'vscode-resource' });
-        const resizingScript = vscode.Uri.file(path.join(extensionPath, 'commands', 'azureCommands', 'acr-logs-utils', 'resizable.js')).with({ scheme: 'vscode-resource' });
-        const styleFile = vscode.Uri.file(path.join(extensionPath, 'commands', 'azureCommands', 'acr-logs-utils', 'style', 'stylesheet.css')).with({ scheme: 'vscode-resource' });
-        const iconStyle = vscode.Uri.file(path.join(extensionPath, 'commands', 'azureCommands', 'acr-logs-utils', 'style', 'fabric-components', 'css', 'vscmdl2-icons.css')).with({ scheme: 'vscode-resource' });
+        const scriptFile = vscode.Uri.file(path.join(ext.context.extensionPath, 'commands', 'azureCommands', 'acr-logs-utils', 'logScripts.js')).with({ scheme: 'vscode-resource' });
```

Comment:
nit for future: Personally I'd rather see ext.context.extensionPath pulled out into a variable like the original code because:
1) The diff is much smaller
2) It's easier to see the common code
3) [very minor] perf improvements over looking up property contents each time

Don't both changing it now, though.

Reply:
sure? It takes me less than a minute.

Reply:
Up to you.  I think you already changed it, so I'll resolve.

Reply:
Ah, you resolved. :-)